### DISCUSSION TITLE
x-sort:item does not support string id's

### DISCUSSION THREAD

**Discussion by titonova** [alpinejs/alpine#4489] [1. Help]
Simply put, this is invalid:

`<div x-sort:item="ab123-cdefg-h1adegf">`

While this is valid:

`<div x-sort:item="1">`

I assume it is not a bug but a core functionality of the sort plugin. In that case, I propose mentioning it within the docs.

Thanks!

**Comment by SimoTod** [comment]
You are missing quotes for your string:
`<div x-sort:item="'ab123-cdefg-h1adegf'">`

It's the same as x-text and other directives.

**Reply by titonova** [reply]
That's correct , silly mistake of mine,. Thanks
